AppCenterDataStorage/AppCenterDataStorage/Internal/Ingestion/MSStorageIngestion.h

```diff
@@ -0,0 +1,33 @@
+#import <Foundation/Foundation.h>
+#import "MSHttpIngestion.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * The header name for update token.
+ */
+static NSString *const kMSHeaderUpdateApiToken = @"x-api-token";
```

Comment:
Move it out of the header if possible. `static` means that it will create a separate field for each file where it will be imported.